awx/main/tasks/jobs.py

```diff
@@ -401,6 +402,9 @@ def acquire_lock(self, project, unified_job_id=None):
                     logger.error("I/O error({0}) while trying to aquire lock on file [{1}]: {2}".format(e.errno, lock_path, e.strerror))
                     raise
                 else:
+                    if not(emitted_lockfile_log):
```

Comment:
super minor but can we change this to `not emitted_lockfile_log` otherwise it looks like a function call

Reply:
no problem! changed!